test/asyncio_tests/test_asyncio_change_stream.py

```diff
@@ -107,7 +107,7 @@ def target():
         change_stream = coll.watch([{'$project': {'_id': 0}}])
         future = change_stream.next()
         self.wait_and_insert(change_stream)
-        with self.assertRaises(InvalidOperation):
+        with self.assertRaises((InvalidOperation, OperationFailure)):
```

Comment:
Might as well do the same in test_motor_change_stream.py, although we're not running those tests yet.